.storybook/polaris-readme-loader.js

```diff
@@ -187,52 +141,53 @@ function parseCodeExamples(data, file) {
     .split('###');
 
   const [, ...allExamples] = examplesAndHeader;
-  const filePath = file.split('polaris-react/').slice(-1)[0];
 
   if (allExamples.length === 0) {
-    throw new Error(
-      chalk`🚨 {red [${matter.data.name}]} No examples found in ${filePath}`,
-    );
+    console.log(chalk`🚨 {red [${
+      matter.data.name
+    }]} No examples found. For troubleshooting advice see https://github.com/Shopify/polaris-react/blob/master/documentation/Component%20READMEs.md#troubleshooting',
```

Comment:
```suggestion
    }]} No examples found. For troubleshooting advice see https://github.com/Shopify/polaris-react/blob/master/documentation/Component%20READMEs.md#troubleshooting
```

Reply:
Tweaked this in a slightly different way, so it is more consistant with how we format our other console.logs